### PR TITLE
Change barbuddy form to allow selection of multiple buddies

### PR DESCRIPTION
**Pull request by ARBaart**

### CODE REVIEW

File: components/bar_buddies/BarBuddyForm.vue [hunk 1]
```diff
@@ -39,16 +39,15 @@ nl:
         <FormInput v-model="form.pronouns" :placeholder="$t('forms.placeholder.pronouns')" />
         <FormValidation name="pronouns" :errors="validationErrors" />
       </FormElement>
-      <FormElement :label="$t('forms.label.barbuddy')" required="true">
-        <FormRadio v-model="form.barbuddy" :label="$t('forms.label.languages.no_preference')" option="no_preference" />
-        <FormRadio
+      <FormElement :label="$t('forms.label.barbuddy')" required="false">
+        <FormMultiCheckbox
```
Comment (casperboone) [inline components/bar_buddies/BarBuddyForm.vue:43]: I think you forgot to include this component in your commit

**Review by laurensWe — CHANGES_REQUESTED** [review]
image

De mogelijkheid om met iemand af te spreken ontbreekt nu eigenlijk helemaal :(

File: components/bar_buddies/BarBuddyForm.vue (hunk 1 above)
Comment (ARBaart) [reply components/bar_buddies/BarBuddyForm.vue:43]: I did do that

**Comment by ARBaart** [comment]
Deprecated as the barbuddy form has already been rewritten for the new version

**Pull request closed without merging**